Snowflake.Data.Tests/IntegrationTests/SFMaxLobSizeSwitchIT.cs

```diff
@@ -0,0 +1,51 @@
+using System.Data;
+using NUnit.Framework;
+using Snowflake.Data.Client;
+
+namespace Snowflake.Data.Tests.IntegrationTests
+{
+    [TestFixture]
+    public class SFMaxLobSizeSwitchIT : SFBaseTest
+    {
+        [Test]
+        [Ignore("TODO: Enable when Max LOB size is available on the automated tests environment")]
+        public void TestIncreaseMaxLobSizeParameterSwitchSelect()
+        {
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+                IDbCommand cmd = conn.CreateCommand();
+
+                //setup
```

Comment:
maybe extract it into a method since the comment is needed for a block of code?

Reply:
I don't see much value here, would decrease readability.